enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/components/SdkIframeEmbedRoute.tsx

```diff
@@ -78,6 +79,8 @@ const SdkIframeEmbedView = ({
 }: {
   settings: SdkIframeEmbedSettings;
 }): ReactNode => {
+  const rerenderKey = useParamRerenderKey(settings);
```

Comment:
Currently `initialSqlParameters` and `initialParameters` are not updating properly as it is "initial", so we need to forcibly re-mount the question & dashboard with a `key` prop.